Sources/StreamChat/Database/StreamChatModel.xcdatamodeld/StreamChatModel.xcdatamodel/contents

```diff
@@ -1,20 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="17709" systemVersion="20C69" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
     <entity name="AttachmentDTO" representedClassName="AttachmentDTO" syncable="YES">
-        <attribute name="actions" optional="YES" attributeType="Binary"/>
-        <attribute name="author" optional="YES" attributeType="String"/>
-        <attribute name="extraData" optional="YES" attributeType="Binary"/>
+        <attribute name="data" optional="YES" attributeType="Binary"/>
```

Comment:
Shouldn't we introduce Migration policy after changing CDModel structure?

Reply:
We were discussing this some time ago and decided that we will not do migrations but the database will be flushed when there will be some breaking changes. 
Really good catch cause I'm not sure we tested this logic.
Created ticket for that.